swan-lake/learn/tools-ides/setting-up-visual-studio-code/language-intelligence.md

```diff
@@ -52,11 +52,19 @@ These allow you to perform the below tasks easily based on the diagnostics and t
 - Add missing imports 
 - Create variable definitions
 - Create an undefined function
+- Map record types automatically
 
-For example, you can add documentation for a function as shown below.
+For another example, you can add documentation for a function as shown below.
 
  ![Code actions](/swan-lake/learn/images/code-actions.gif)
 
+For example, you can use the automatic data mapping code action as shown below.
+ > **Note**: By default, the automatic data mapping code action is disabled. To enable it, click **Settings**, select **Ballerina** from **Extensions**, and select the **Data Mapper: Enabled** checkbox. (If you are using MacOS, you can go to **Settings** by pressing `Cmd` + `,`.)
```

Comment:
Why do we need to repeat the sentence ` (If you are using MacOS, you can go to **Settings** by pressing `Cmd` + `,`.` here and as a `Tip` below.

Reply:
The review comment was to remove the tip and add this in brackets.

Reply:
With the new commit the repeated sentence was removed